android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/ssdp/Ssdp.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright 2014 Fraunhofer FOKUS
```

Comment:
Our open source team has informed us that this should read:

Copyright 2014-2015 The Physical Web Authors